src/Razor/src/Microsoft.NET.Sdk.Razor/build/netstandard2.0/Sdk.Razor.CurrentVersion.targets

```diff
@@ -336,15 +337,10 @@ Copyright (c) .NET Foundation. All rights reserved.
 
   <!-- Determine what compiler and versions of the language to target. For 2.x targeting projects, these are carried by packages referenced by the project. Use this -->
 
-  <!-- Use the 2.x compiler if we're building an application that references Microsoft.AspNetCore.Razor.Design. -->
-  <Import Project="$(RazorCodeGenerationTargetsPath)"
```

Comment:
This is the magic right? ALl the rest is just refactoring?

Reply:
Yup. We just ignore the targets / compiler from Razor.Design package and use the one from the SDK. A similar change was made in 6.0 and we've tested the change locally + CTI scenarios (VS 2022 + .NET 2.1 Framework apps). This change is motivated by this - https://devblogs.microsoft.com/dotnet/net-core-2-1-will-reach-end-of-support-on-august-21-2021